assets/src/scripts/_form-setup.js

```diff
@@ -1,12 +1,12 @@
 import { effect } from "@preact/signals";
-import { reviewUrl } from "./_data";
+import { releaseUrl } from "./_data";
```

Comment:
Hmm, what if they reject all outputs. It's not a release then.

I think the original reviewUrl is the better name, tbh

Reply:
At the time of writing, rejecting all outputs didn't allow you to call this endpoint. I also don't think this endpoint is reviewing anything, it's always going to do a release of some sort. However, it's not that important, so I'll remove that commit.